corehq/apps/geospatial/views.py

```diff
@@ -475,6 +476,12 @@ def post(self, request, domain, *args, **kwargs):
             for child_case_id, parent_case_id in child_to_parent_case_id.items():
                 self._add_related_case(case_id_to_owner_id, child_case_id, parent_case_id)
 
+        if len(case_id_to_owner_id) > self.ASYNC_CASES_LIMIT:
+            return HttpResponseBadRequest(
+                _("Max limit for cases to be reassigned including related cases exceeded."
+                  " Please select a lower value to update at time or reach out to support")
```

Comment:
Nit: You can also suggest that the user uncheck the "Include related cases" checkbox as this would put the case count within the limit.

Reply:
Good idea. 
Added in https://github.com/dimagi/commcare-hq/pull/35037/commits/7c48b07482a049d34714c961364eda889d86db42